config/locales/idv/en.yml

```diff
@@ -169,25 +169,32 @@ en:
         other: You have <strong>%{count} attempts remaining.</strong>
       button:
         fail: Okay
+        jobfail: Try again tomorrow
         warning: Try again
       financials:
         fail: Your account is <strong>locked for 24 hours</strong> before you can
           try to verify again. Check our Help section for more information about identity
           verification.
         heading: We could not find records matching your financial information.
+        jobfail: Something went wrong and we cannot process your request at this time.
+          Please try again tomorrow.
         timeout: Our request to verify your information timed out.
```

Comment:
Totally unrelated to this PR, but I think we should talk to a content person about this message. I think many users may not really know what "timed out" means, and it doesn't give them any hints on what to do now.

Reply:
Yes, all of our text is/should be reviewed by content folks. It's possible that this particular entry hasn't been reviewed yet.